modules/service.sh

```diff
@@ -19,20 +12,37 @@ service_enable() {
     local service="$1"
     local token="$2"
 
-    check_user
-    service_check_support "$service"
-    _service_check_enabled "$service" || exit 6
+    _service_check_user
+    _service_check_support "$service"
+    _service_check_enabled "$service"  || exit 6
     "${service}_validate_token" "$token" || exit 3
     "${service}_enable" "$token"
 }
 
+service_disable() {
+    local service="$1"
+
+    _service_check_user
+    _service_check_support "$service"
+    _service_check_disabled "$service"
```

Comment:
Like we did for service-already-enabled, should we add a "|| exit 8"' here if the service is already disabled, and add this new exit code to the manpage? ("8" was the next available one)